hw/ip_templates/rv_plic/fpv/tb/waivers.tcl

```diff
@@ -52,3 +52,18 @@ assert -name InpAlwaysOnehot0_A {$onehot0(dut.u_reg.u_prim_reg_we_check.u_prim_o
 # Refer to waivers.tcl for the detailed argument as why this is not possible.
 assert -name RvPlicRegNoOneHot&EnError \
   {dut.u_reg.u_prim_reg_we_check.u_prim_onehot_check.err_o == 0}
+
+# Jasper analyzed this precondition as unreachable. It requires err_o in u_prim_onehot_check
```

Comment:
I agree, although you can probably say it much more briefly if you're happy to be a bit lazier :-) I'd also link it to the previous assertion explicitly.

Maybe something like "Jasper concluded that this precondition is unreachable. It would need a one-hot error, which the design will never generate. The check is to detect fault injection, which isn't tested in FPV at the moment. The assertion added above requires Jasper to prove that this can't happen."